metabrainz/reports/annual_reports/files/2018/content.html

```diff
@@ -0,0 +1,29 @@
+<h2>Archive notice</h2>
+
+<p>
+Annual reports from 2006 to 2017 are still available for history.
+There was no annual report in this form before 2006.
+Since 2018, no annual report has been produced as it was too much overhead for one person.
```

Comment:
I'm not sure what would be good here, but this seems like giving excuses rather than saying we just don't do this anymore.

Reply:
If you’re not sure, I’m pretty sure that’s good. As we discussed about it with @mayhem it appeared to be the main reason these reports have been indefinitely suspended. Explaining why there is less transparency on our activity at the moment seems to be the bare minimum.

Reply:
```suggestion
Since 2018, no annual report has been produced as there wasn’t enough interest into it.
```